### PR TITLE
[IMP] project: improve the UX of the app

### PR DESCRIPTION
**Pull request by cedricpri**
Generic UX improvements for the project app.

task-2501760
See odoo/enterprise#18819

### CODE REVIEW

File: addons/web/static/src/legacy/js/views/calendar/calendar_popover.js [hunk 1]
```diff
@@ -206,10 +207,11 @@ var CalendarPopover = Widget.extend(WidgetAdapterMixin, StandaloneFieldManagerMi
 
                 var $field = $('<li>', {class: fieldClass});
                 var $fieldLabel = $('<strong>', {class: 'mr-2', text: _.str.sprintf('%s : ', field.string)});
+
                 $fieldLabel.appendTo($field);
                 var $fieldContainer = $('<div>', {class: 'flex-grow-1'});
                 $fieldContainer.appendTo($field);
-
+
```
Comment (SimonGenin) [inline addons/web/static/src/legacy/js/views/calendar/calendar_popover.js:214]: All the changes in this file are unnecessary diffs.

File: addons/web/static/src/legacy/xml/kanban.xml [hunk 2]
```diff
@@ -20,12 +20,12 @@
...
                         <t t-if="widget.grouped_by_m2o">
-                            <a t-if="widget.editable and widget.id" role="menuitem" class="dropdown-item o_column_edit" href="#">Edit Stage</a>
+                            <a t-if="widget.editable and widget.id" role="menuitem" class="dropdown-item o_column_edit" href="#">Edit</a>
                             <a t-if="widget.deletable and widget.id" role="menuitem" class="dropdown-item o_column_delete" href="#">Delete</a>
                         </t>
                         <t t-if="widget.has_active_field and widget.archivable">
-                            <a role="menuitem" href="#" class="dropdown-item o_column_archive_records">Archive All</a>
-                            <a role="menuitem" href="#" class="dropdown-item o_column_unarchive_records">Unarchive All</a>
+                            <a role="menuitem" href="#" class="dropdown-item o_column_archive_records">Archive All Tasks</a>
+                            <a role="menuitem" href="#" class="dropdown-item o_column_unarchive_records">Unarchive All Tasks</a>
```
Comment (SimonGenin) [inline addons/web/static/src/legacy/xml/kanban.xml:28]: Are you sure it's what you want to do ? You change the xml of the kanban view globally here, meaning that your change will impact all the kaban views in any modules.
If you want to only do it in your module, you'll need to overwrite this template instead :)

**Review by SimonGenin — CHANGES_REQUESTED** [review]

**Review by xavierbol — CHANGES_REQUESTED** [review]
Thanks for your work ! 🙂 
I have made some comments.

File: addons/hr_timesheet/views/project_views.xml [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+        <field name="arch" type="xml">
+            <pivot string="Project Tasks" sample="1" js_class="project_pivot">
+                <field name="total_hours_spent" widget="float_time"/>
+                <field name="overtime" widget="float_time"/>
+                <field name="remaining_hours" widget="float_time"/>
+                <field name="effective_hours" widget="float_time"/>
+                <field name="subtask_effective_hours" widget="float_time"/>
+            </pivot>
+        </field>
+    </record>
```
[the remark this answers is not in the document]
Comment (cedricpri) [reply addons/hr_timesheet/views/project_views.xml:17]: Sorry, I didn't see that but it seems that another commit actually introduced the same feature while I was working on them, and a rebase then duplicated the view. I removed this.

**Review by xavierbol — APPROVED** [review]
LGTM ! Thanks for the changes. 🙂

**Review by IT-Ideas — COMMENTED** [review]
Thanks for your work 🙂 
Just some little things

**Review by IT-Ideas — APPROVED** [review]
LGTM Thanks

**Comment by IT-Ideas** [comment]
@robodoo r+

**Pull request closed without merging**